code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm

```diff
@@ -71,10 +71,6 @@
 		if(prob(80))
 			var/atom/throw_target = get_edge_target_turf(L, dir)
 			L.throw_at(throw_target, rand(1,2), 7, src)
-			if(diseased)
```

Comment:
Can the diseased variable be removed, or is it used elsewhere?

Reply:
Yeah, you are totally right, removed!